### PR TITLE
Update CI Tests

### PR DESCRIPTION
**Pull request by karljj1**
- Fixed python error when using unity-downloader
- Added support for 2019.3, 2020.1 and 2020.2.
- Fixed failing XR tests on 2019.3+

- Disabled Windows Mouse tests as they wont work in batchmode. We can fix this later

### CODE REVIEW

File: Packages/manifest.json [hunk 1]
```diff
@@ -1,13 +1,7 @@
 {
   "dependencies": {
-    "com.unity.ads": "2.0.8",
-    "com.unity.analytics": "3.3.2",
-    "com.unity.cinemachine": "2.3.4",
-    "com.unity.multiplayer-hlapi": "1.0.2",
     "com.unity.package-manager-doctools": "1.1.1-preview.5",
     "com.unity.package-manager-ui": "2.1.2",
-    "com.unity.probuilder": "4.0.5",
```
Comment (Rene-Damm) [inline Packages/manifest.json:9]: This one's needed for TouchSamples

File: Packages/manifest.json [hunk 2]
```diff
@@ -1,13 +1,7 @@
 {
   "dependencies": {
-    "com.unity.ads": "2.0.8",
-    "com.unity.analytics": "3.3.2",
-    "com.unity.cinemachine": "2.3.4",
```
Comment (Rene-Damm) [inline Packages/manifest.json]: Not sure about this one... is it used by TouchSamples?

File: Packages/manifest.json [hunk 3]
```diff
@@ -1,13 +1,7 @@
 {
   "dependencies": {
-    "com.unity.ads": "2.0.8",
-    "com.unity.analytics": "3.3.2",
```
Comment (Rene-Damm) [inline Packages/manifest.json]: Don't we need that for UNITY_ANALYTICS?

File: Packages/manifest.json (hunk 1 above)
Comment (karljj1) [reply Packages/manifest.json:9]: Ah. It causes compilation issues in 2019.3+
Maybe It just needs upgrading to a new version.

File: ProjectSettings/EditorSettings.asset [hunk 4]
```diff
@@ -20,3 +20,4 @@ EditorSettings:
   m_CollabEditorSettings:
     inProgressEnabled: 1
   m_EnableTextureStreamingInPlayMode: 1
+  m_AssetPipelineMode: 1
```
Comment (karljj1) [inline ProjectSettings/EditorSettings.asset:23]: Suppresses the popup when running outside of batchmode

**Review by Rene-Damm — APPROVED** [review]

**Pull request merged**